### PR TITLE
Update snowflake-configs.md

### PR DESCRIPTION
**Pull request by amychen1776**
update docs to be more clear

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-amychen1776-patch-1-dbt-labs.vercel.app/reference/resource-configs/snowflake-configs

### CODE REVIEW

File: website/docs/reference/resource-configs/snowflake-configs.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ Snowflake allows two configuration scenarios for scheduling automatic refreshes:
 - **Time-based** &mdash; Provide a value of the form `<int> { seconds | minutes | hours | days }`. For example, if the dynamic table needs to be updated every 30 minutes, use `target_lag='30 minutes'`.
 - **Downstream** &mdash; Applicable when the dynamic table is referenced by other dynamic tables. In this scenario, `target_lag='downstream'` allows for refreshes to be controlled at the target, instead of at each layer.
 
-Learn more about `target_lag` in Snowflake's [docs](https://docs.snowflake.com/en/user-guide/dynamic-tables-refresh#understanding-target-lag).
+Learn more about `target_lag` in Snowflake's [docs](https://docs.snowflake.com/en/user-guide/dynamic-tables-refresh#understanding-target-lag). Please note that Snowflake does not support a target lag of ' > 1 minute'.
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/snowflake-configs.md]:
```suggestion
Learn more about `target_lag` in Snowflake's [docs](https://docs.snowflake.com/en/user-guide/dynamic-tables-refresh#understanding-target-lag). Please note that Snowflake supports a target lag of 1 minute or longer.
```

**Review by mirnawong1 — APPROVED** [review]
hey @amychen1776 ! looks great,thank you! and just a small sugg on my end. approved otherwise!

File: website/docs/reference/resource-configs/snowflake-configs.md (hunk 1 above)
Comment (amychen1776) [reply website/docs/reference/resource-configs/snowflake-configs.md]: This is wrong :) Snowflake doesn't support anything smaller than 1 minute

Comment (mirnawong1) [reply website/docs/reference/resource-configs/snowflake-configs.md]: oh ok sorry!  what about @amychen1776 ?

Please note that Snowflake supports a target lag of 1 minute or longer?

Comment (amychen1776) [reply website/docs/reference/resource-configs/snowflake-configs.md]: That works! Feel free to update and merge :)

**Pull request merged**